README.md

```diff
@@ -1,21 +1,34 @@
 # cloudflare-worker-template
-Bootstrap your Cloudflare Worker to connect to a pgEdge database
+Bootstrap your Cloudflare Worker to connect to a pgEdge database.
 
 [![Deploy to Cloudflare Workers](https://deploy.workers.cloudflare.com/button)](https://deploy.workers.cloudflare.com/?url=https://github.com/pgEdge/cloudflare-worker-template)
 
 This is a simple Cloudflare Worker that connects to a pgEdge database. Use it as a starting point 
-for your own project. 
+for your own project! Simply click on the button above to deploy this worker to your Cloudflare
+account and create a Github repository for it.
 
 ## pgEdge connection
 
-To find your pgEdge connection string, go to the [pgEdge console](https://app.pgedge.com). If you have 
-not yet signed up for pgEdge, you can easily sign in with your social accounts. Creating a new database 
-is free and takes less than a minute.
+The easiest way to get your worker connected to a pgEdge database is to go to the [pgEdge console](https://app.pgedge.com).
+If you have not yet signed up for pgEdge, you can easily sign in with your social accounts. Creating a new 
+database is free and takes less than a minute.
 
-Once you've reached the console, navigate to the database you want to connect to. In the `Connect to your database` 
-section, you will find your username, password, and the host is the `Domain`.  
+Once you've connected the console, navigate to the database you want to connect to. On the sidebar, click
+on the "Integrations" tab. You should see a "Cloudflare Workers" integration. This will provide you with
+all the details you need to connect your worker to your database.
 
-## DB connection
+If you prefer to follow READMEs instead of clicking around, here are the steps to get your worker connected...
+
+### Setup your local environment
+
+Once you've deployed your worker and cloned your repository, you'll need to setup your local environment to 
+be able to run the worker locally. You'll need to install wrangler, the Cloudflare CLI tool.
```

Comment:
It could be worth adding a link to the wrangler docs on the word `wrangler`. I ended up searching for them, because I was unfamiliar with this tool.

Reply:
This is a good suggestion - I'll add it.